test/functions/iterators/recursive/follow-record-updates.chpl

```diff
@@ -43,7 +43,7 @@ proc report(const ref REPO: RRR) {
 /////////////////////////////////
 
 // The former bug was not seen if we add 'const ref' to 'ARG'.
-proc WriteThis(ARG: RRR) {
+proc displayThis(ARG: RRR) {
```

Comment:
Out of curiosity: is this only a stylistic change? Do we have something preventing us from using `WriteThis` name? With that capitalization?

Reply:
It's stylistic, because I consider naming the routine `WriteThis` to be bad form (for what I would hope are fairly obvious reasons).